mob.go

```diff
@@ -45,28 +45,31 @@ func doneSquash(value string) string {
-	VoiceCommand                   string // override with MOB_VOICE_COMMAND
-	VoiceMessage                   string // override with MOB_VOICE_MESSAGE
-	NotifyCommand                  string // override with MOB_NOTIFY_COMMAND
-	NotifyMessage                  string // override with MOB_NOTIFY_MESSAGE
-	NextStay                       bool   // override with MOB_NEXT_STAY
-	StartIncludeUncommittedChanges bool   // override with MOB_START_INCLUDE_UNCOMMITTED_CHANGES variable
-	StashName                      string // override with MOB_STASH_NAME
-	WipBranchQualifier             string // override with MOB_WIP_BRANCH_QUALIFIER
-	WipBranchQualifierSeparator    string // override with MOB_WIP_BRANCH_QUALIFIER_SEPARATOR
-	WipBranchPrefix                string // override with MOB_WIP_BRANCH_PREFIX
-	DoneSquash                     string // override with MOB_DONE_SQUASH
-	Timer                          string // override with MOB_TIMER
-	TimerRoom                      string // override with MOB_TIMER_ROOM
-	TimerLocal                     bool   // override with MOB_TIMER_LOCAL
-	TimerRoomUseWipBranchQualifier bool   // override with MOB_TIMER_ROOM_USE_WIP_BRANCH_QUALIFIER
-	TimerUser                      string // override with MOB_TIMER_USER
-	TimerUrl                       string // override with MOB_TIMER_URL
+	CliName                        string       // override with MOB_CLI_NAME
+	RemoteName                     string       // override with MOB_REMOTE_NAME
+	WipCommitMessage               string       // override with MOB_WIP_COMMIT_MESSAGE
+	GitHooksEnabled                bool         // override with MOB_GIT_HOOKS_ENABLED
+	RequireCommitMessage           bool         // override with MOB_REQUIRE_COMMIT_MESSAGE
+	VoiceCommand                   string       // override with MOB_VOICE_COMMAND
+	VoiceMessage                         string // override with MOB_VOICE_MESSAGE
+	NotifyCommand                        string // override with MOB_NOTIFY_COMMAND
+	NotifyMessage                        string // override with MOB_NOTIFY_MESSAGE
+	NextStay                             bool   // override with MOB_NEXT_STAY
+	StartIncludeUncommittedChanges       bool   // override with MOB_START_INCLUDE_UNCOMMITTED_CHANGES variable
+	StashName                            string // override with MOB_STASH_NAME
+	WipBranchQualifier                   string // override with MOB_WIP_BRANCH_QUALIFIER
+	WipBranchQualifierSeparator          string // override with MOB_WIP_BRANCH_QUALIFIER_SEPARATOR
+	WipBranchPrefix                      string // override with MOB_WIP_BRANCH_PREFIX
+	DoneSquash                           string // override with MOB_DONE_SQUASH
+	WriteLastModifiedFileInCommitMessage bool   // override with MOB_WRITE_LAST_MODIFIED_FILE_IN_COMMIT_MESSAGE
+	OpenLastModifiedFile                 bool   // override with MOB_OPEN_LAST_MODIFIED_FILE
+	OpenCommand                          string // override with MOB_OPEN_COMMAND
```

Comment:
I think setting the MOB_OPEN_COMMAND to a nonempty string should enable the open feature.

Reply:
Sounds like a good idea :)

Reply:
done